dashmips/debugger.py

```diff
@@ -92,8 +92,10 @@ def checkfile(f: str) -> bool:
                 break
 
         is_known_file = any(checkfile(fn) for fn in program.filenames)
+        remote_breakpoints.append(breakpoint)
         if real_line is not None and is_known_file:
-            remote_breakpoints.append(breakpoint)
             local_breakpoints.append(real_line)
+        elif real_line is None and is_known_file:
+            local_breakpoints.append(-1)
```

Comment:
Let's document this, right here in a comment what -1 means and maybe a note in the wiki